docs/glossary.md

```diff
@@ -0,0 +1,106 @@
+**Contet Management System (CMS)**:
+The Opin Platform consists of the platform itself, meaning the part where users can add, edit, delete organisations, projects, modules, phases, ideas, comments, ratings, reports etc. as well as of descriptive webpages that give more information on the platform, such as imprint, help and how to's. As these webpages should be editable by a user a Content Management System (CMS) is used. For this project the Wagtail CMS used. Major advantage of this CMS is that it builds up upon the Webframework Django which is also used for the OPIN Platform. Because of that it is possible, that users who are logged in on the platform and also have rights to access the CMS-Administration interface, do not have to log in again. 
+
+---
+**Comment**:
+A comment is the expressed opinion of a user. Users on Opin can comment on ideas, documents and paragraphs. 
+
+---
+**Creator**
+User geneared content has to be created by an user (usally participant, but other roles possible). If an user has created content, she will assume the creator role for that content. Creators have special rights, because they can delete or modify their own content.
+
+---
+**Dashboard**:
+The part of the opin interface, used by organisation owners, initators and moderators to manage projects on the platform. In the dashboard also users can edit their personal settings, such as username, email-adress, avatars. 
+
+---
+**Feature,Tool, or Method**:
+A feature/method/tool is a formalized way to collaborate in order to reach a common goal. In one participation project, content may be transferred from one feature/method/tool to another or even worked on with different features/methods/tools at the same time (e.g. online and offline)
+
```

Comment:
Hey, i would focus on feature only. In my perspective, "method" is used in f2f-participation (e.g. round table, World Café) that are difficult to implement online. About "tool", in my view, this could refer to OPIN as such: a tool for online participation. Equally to "toggle" as a tool for time-tracking. My five cents. Thanks!